GitUI/CommandsDialogs/BrowseDialog/FormCommitCount.cs

```diff
@@ -28,69 +35,113 @@ private void cbIncludeSubmodules_CheckedChanged(object sender, EventArgs e)
 
         private void FetchData()
         {
-            Loading.Visible = true;
+            ThreadHelper.ThrowIfNotOnUIThread();
+
+            _backgroundLoaderTokenSource.Cancel();
+            _backgroundLoaderTokenSource = new CancellationTokenSource();
```

Comment:
📝 Obvious candidate for `CancellationTokenSequence`

Reply:
This was the change that gave me the idea for that class.